_posts/2023-03-21-announcing-ci.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "US-RSE has a new Fiscal Sponsor!"
```

Comment:
```suggestion
title: "US-RSE Has a New Fiscal Sponsor!"
```

Reply:
I realize this is nitpicky, but the mixed capitalization looks weird to me!

Reply:
Usually, I'm the one making those nitpicky changes, so thanks! 😆